master/buildbot/buildslave/__init__.py

```diff
@@ -75,6 +75,7 @@ def __init__(self, name, password, max_builds=None,
         self.password = password
         self.friendly_name = friendlyName
         self.eid = eid  # External ID
+        self.fqdn = fqdn
```

Comment:
To quote Daniel when I raised the same issue: 
> and FQDN is very descriptive
I'm inclined to agree with him. FQDN is a common enough acronym, and it should be pretty clear that it's referring to the slave's domain since it's a parameter of the slave object.

I'll update the documentation now.